### PR TITLE
Add Plugdev Permissions for Openocd to Connect to Nucleo

### PR DESCRIPTION
**Pull request by jonathanlew**
### Description
Add 60-openocd.rules to udev for permissions for using openocd to connect to nucleo board

### Testing Done
Manual

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [ ] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

File: environment_setup/install_openocd.sh [hunk 1]
```diff
@@ -21,6 +21,8 @@ git clone https://github.com/ntfreak/openocd.git $OPENOCD_TMP_DIR
 
 # Build and Install Openocd
 cd $OPENOCD_TMP_DIR
+sudo cp contrib/60-openocd.rules /etc/udev/rules.d/
+sudo adduser $USER plugdev
```
Comment (garethellis0) [inline environment_setup/install_openocd.sh:25]: Could we move this into a section above called `# Set permissions to allow this user to access USB devices`?

**Review by garethellis0 — CHANGES_REQUESTED** [review]

File: environment_setup/install_openocd.sh (hunk 1 above)
Comment (jonathanlew) [reply environment_setup/install_openocd.sh:25]: done

**Review by garethellis0 — APPROVED** [review]
LGTM :+1: :tada:

**Review by henryymliu — APPROVED** [review]
🚢🤡

**Pull request merged**